### PR TITLE
Loading optimizations and other fixes 

### PR DESCRIPTION
**Pull request by lassejaco**
## Description
- Rework threejs asset loading for smoother transitions between pages
- Proper teardown of threejs on page nav (to enable inline navigation / remove need for full reload on page navigation)
- Add preloading for threejs assets 
- Font loading optimizations
- Nextjs upgraded to latest version
- Remove "mouse down link" hack for cleaner code and fix edge case when navigating with mac shortcuts

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "@types/react-dom": "18.0.9",
     "eslint": "8.28.0",
     "eslint-config-next": "13.1.1",
-    "next": "13.1.1",
+    "next": "14.1.0",
```
[the remark this answers is not in the document]
Comment (lassejaco) [reply package.json:17]: Oh, certainly. Fixed!

[the remark this answers is not in the document]
Comment (lassejaco) [reply package.json:17]: Also - we are still iterating a bit - don't merge just yet :D

[the remark this answers is not in the document]
Comment (pettinarip) [reply package.json:17]: Sounds good! lmk if you need eyes on this when you are done 👍🏼

**Comment by lassejaco** [comment]
Hey @pettinarip  - can we reopen this PR?

**Comment by pettinarip** [comment]
> Hey @pettinarip - can we reopen this PR?

Hey @lassejaco. I'm a little confused. Curious to know what happened here, did I close this without asking you? or did we have a chat about closing it? if the former, let me apologize, it was not the intention. I must have closed it without even realize it.

Reopening 🙏🏼

**Comment by lassejaco** [comment]
Not to worry! It is a very slow moving PR , so I get if it seemed dead :D 

Thanks for reopening! May need your help to merge in sometime next week - I'll let you know :-)

**Comment by lassejaco** [comment]
Going to close and reopen because it seems like the PR isn't picking up my latest commits

**Pull request closed without merging**